tests/ConstraintIB/oscillating_rigid_cylinder.restart=40.input

```diff
@@ -0,0 +1,340 @@
+      }
+   }
+
+   PressureHypreSolver {
+      solver_type           = "PFMG"
+      rap_type              = 0
+      relax_type            = 2
+      skip_relax            = 1
+      num_pre_relax_steps   = 2
+      num_post_relax_steps  = 2
+      relative_residual_tol = 1.0e-2
+      max_iterations        = 100
+      enable_logging        = FALSE
+   }
+
+   PressureFACSolver {
+      num_pre_sweeps  = 0
+      num_post_sweeps = 3
+      coarse_solver_choice = "hypre"
+      coarse_solver_tolerance = 1.0e-12
+      coarse_solver_max_iterations = 1
+      prolongation_method = "LINEAR_REFINE"
+      restriction_method  = "CONSERVATIVE_COARSEN"
+      hypre_solver {
+         solver_type           = "PFMG"
+         rap_type              = 0
+         relax_type            = 2
+         skip_relax            = 1
+         num_pre_relax_steps   = 0
+         num_post_relax_steps  = 3
+         relative_residual_tol = 1.0e-12
+         max_iterations        = 1
+         enable_logging        = FALSE
+      }
+   }
+}
+
+// Oscillating Cylinder
+ConstraintIBKinematics {
+
+     Cylinder {
+     structure_names                  = "cylinder2d"
+     structure_levels                 = MAX_LEVELS - 1
+     calculate_translational_momentum = 0,0,0
+     calculate_rotational_momentum    = 0,0,0
+     lag_position_update_method       = "CONSTRAINT_VELOCITY"
+     tagged_pt_identifier             = MAX_LEVELS - 1, 0  // level, relative idx of lag point
+
+     // Cylinder oscillates with U_infinity * cos(2*pi*freq*t)
+     frequency                        = FREQUENCY
+     U_infinity                       = 2.0
+
+    // Additional prescribed velocity
+     prescribed_trans_velocity        = 0.0
+
+
+    }
+
+}
+
+
+  PrintOutput {
+
+      print_output                = TRUE    //default false, if false nothing will be printed.
+      output_interval             = int(0.001/DT + 0.5)       //default 1
+      output_drag_kinetic_energy  = TRUE    //default false, if false it will not be computed.
+      output_power                = FALSE    //default false, if false it will not be computed.
+      output_rig_transvel         = FALSE    //default false
+      output_rig_rotvel           = FALSE    //default false
+      output_com_coords           = FALSE    //default false
+      output_moment_inertia       = FALSE   //default false
+
+      output_dirname              = "./cylinder_dump"
+      base_filename               = "cylinder2d"
+  }
+
+
+
+
+
+
+
+Main {
+// log file parameters
+   log_file_name    = "IB2d_cylinder2d.log"
+   log_all_nodes    = FALSE
+
+// visualization dump parameters
+   viz_writer            = "VisIt","Silo"
+   viz_dump_interval     = int(0.01 * (1.0/FREQUENCY)/DT  + 0.5)  //zero to turn off
+   viz_dump_dirname      = "viz_cylinder2d"
+   visit_number_procs_per_file = 1
+
+// restart dump parameters
+   restart_interval      = 20                 // zero to turn off
+   restart_write_dirname = "restart_IB2d"
```

Comment:
```suggestion
   restart_write_dirname = "restart"
```
The test suite requires that this directory always have the same name (`"restart"`)